pkg/operator/status.go

```diff
@@ -308,6 +308,13 @@ func (c *Controller) computeStatus(tuned *tunedv1.Tuned, conditions []configv1.C
 			availableCondition.Message = fmt.Sprintf("%v/%v Profiles failed to be applied", numDegradedProfiles, len(profileList))
 		}
 
+		if len(c.tunedProfileConflict) > 0 {
```

Comment:
I know this is cheaper than doing the count over AllProfiles here, but using the direct value instead of a precomputed cache might be easier to maintain.

Reply:
Yes, as I said above, it is cheaper.